src/pages/index.js

```diff
@@ -95,10 +96,16 @@ export default function WRMCWebsite() {
     // TODO: we should create a container component for the Home page
     // (and any other pages that end up having multiple components)
     "Home" : <div>
-              <ShowOTW show={sotw} handleClick={clickShow}/> 
+              <ShowOTW show={sotw} handleClick={clickShow}/>
               <p>{""}</p>
               <DisplayCurrentShow show={isOnAir ? upcomingShows[0] : shows.find(show => show.id === 12345)} handleClick={clickShow}/>
               <p>{""}</p>
+              <DisplayCurrentPlaylist playlist = {currentPlaylist ? currentPlaylist : 
+              {"date": "",
+              id: 12345,
+              showID: 12345}}
```

Comment:
I don't think this actually works as an empty playlist, because it has an `id`, it will still display the songs with the same `playlistID`, if there are any. I think we need to simply pass `undefined` here and deal with it inside the `DisplayCurrentPlaylist` component.

Reply:
I'm wondering if that would still be in line with the discussion about ternary operators in class last week

Reply:
I believe that would be ok, as undefined is still a value. We could also pass in an empty object instead, but it might be harder to check in that case.

Reply:
I just made some changes. Removed the ternary operator and just passed through `currentPlaylist`. As Ethan S mentions, this will pass through undefined values at time. To deal with that I just added an if statement to the `DisplayCurrentPlaylist` component which checks to see if undefined was passed through